requirements.txt

```diff
@@ -1,10 +1,11 @@
-Django==1.11.8
+Django==2.0.2
+djangorestframework==3.7.4
```

Comment:
are we using it yet?

Reply:
No, but wagtail allows a broad range of versions (`djangorestframework>=3.1.3,<4.0`). This pins it to the django 2.0 compat version.
http://www.django-rest-framework.org/topics/release-notes/#374 

Might be an issue to raise with wagtail core.

Reply:
I think it is worth it. I sense a minor release coming in not too far of a future containing all the little tweaks like that